airbyte-cdk/python/airbyte_cdk/sources/streams/concurrent/adapters.py

```diff
@@ -347,35 +342,20 @@ def check_availability(self, stream: Stream, logger: logging.Logger, source: Opt
         return stream_availability.is_available(), stream_availability.message()
 
 
-class StreamAvailabilityStrategy(AbstractAvailabilityStrategy):
+class AlwaysAvailableAvailabilityStrategy(AbstractAvailabilityStrategy):
```

Comment:
Can I request that we define `AlwaysAvailableAvailabilityStrategy` in `streams/concurrent/availability_strategy.py` instead. I'm almost certain that we'll also need to use this in the new `ConcurrentDeclarativeSource` as well. So that might be a better spot if its used in other places.